masterlist.yaml

```
@@ -1769,6 +1769,30 @@ plugins:
       - *alreadyInOrFixedByFCF
       - *alreadyInOrFixedByPRPv0_59
 
+  - name: '(MTM-NukaTownEastArchwayFix|MTM-GardenTerraceCeilingFixedByMonorail)\.es[lp]'
```

This regex can be simplified a bit:

```suggestion
  - name: 'MTM-(NukaTownEastArchwayFix|GardenTerraceCeilingFixedByMonorail)\.es[lp]'
```